appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.task;
+
+// [START cloud_tasks_worker]
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// The Worker servlet should be mapped to the "/worker" URL.
+// With @WebServlet annotation the webapp/WEB-INF/web.xml is no longer required.
+@WebServlet(
+    name = "TaskWorker",
+    description = "Task handler",
+    urlPatterns = "/taskqueues/worker"
+)
+public class Worker extends HttpServlet {
+
+  private static final Logger log = Logger.getLogger(Worker.class.getName());
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    String key = request.getParameter("key");
+
+    // Do something with key.
+    log.info("Worker is processing " + key);
```

Comment:
I don't understand what this Servlet is supposed to be doing. Can you add a comment explaining what the goal of this endpoint is and how to plays into the larger app?

Reply:
I updated the servlet description. Is it descriptive enough?

Reply:
Made a suggestion. PTAL.